packages/ra-core/src/controller/field/useReferenceManyFieldController.ts

```diff
@@ -163,7 +163,7 @@ export const useReferenceManyFieldController = <
         },
         {
             enabled: get(record, source) != null,
-            keepPreviousData: true,
```

Comment:
I searched the reason for this change, so I put it here for reference: https://tanstack.com/query/v5/docs/react/guides/migrating-to-v5#removed-keeppreviousdata-in-favor-of-placeholderdata-identity-function

Note that our docs mention `keepPreviousData` here and there, so its needs to be updated, too.

Reply:
Documentation will be updated in a dedicated PR